rules/README.md

```diff
@@ -1,6 +1,6 @@
 ## Rules
 
-The following rules are provided by `@foxglove/eslint-plugin`. You may need to enable the plugin by adding `@foxglove` in the `plugins` section of your eslint configuration.
```

Comment:
I would mantain this comment, to enable plugin adding "@lichtblick" in the eslint config.